app/main.py

```diff
@@ -1 +1,69 @@
-# write your code here
+from typing import List
+
+
+class SoftwareEngineer:
+    def __init__(self, name: str) -> None:
+        self.name = name
+        self.skills: List[str] = []
+
+    def learn_skill(self, skill: str) -> None:
+        if skill not in self.skills:
+            self.skills.append(skill)
+
+
+class FrontendDeveloper(SoftwareEngineer):
+    DEFAULT_SKILLS = ["JavaScript", "CSS", "HTML"]
+
+    def __init__(self, name: str) -> None:
+        super().__init__(name)
+        self.skills = self.DEFAULT_SKILLS.copy()
```

Comment:
```suggestion
class FrontendDeveloper(SoftwareEngineer):
    def __init__(self, name: str) -> None:
        super().__init__(name)
        self.skills += ["JavaScript", "CSS", "HTML"] 
```
In this case, creating the variable ** DEFAULT_SKILLS** is unnecessary.